lib/package.json

```diff
@@ -73,6 +73,7 @@
   },
   "dependencies": {
     "buffer-crc32": "^1.0.0",
+    "compare-versions": "^6.1.1",
```

Comment:
I tried adding "semver" initially but it doesn't play nicely with modules in some test cases
```
import { lt as semverLt } from 'semver';
🚧 Browser logs:
SyntaxError: The requested module './../../node_modules/semver/index.js' does not provide an export named 'lt'

```